tools/ansible/tasks/wsu/main.yaml

```diff
@@ -2,7 +2,7 @@
 - hosts: localhost
   vars:
     project_root: "{{ playbook_dir }}/../../../.."
-    kubelet_location: "https://dl.k8s.io/v1.14.0/kubernetes-node-windows-amd64.tar.gz"
+    kubelet_location: "https://dl.k8s.io/v1.16.2/kubernetes-node-windows-amd64.tar.gz"
```

Comment:
@suhanime - Let me know, if you want this to be removed and you will address it in a separate PR

Reply:
This should be removed from here and be introduced in a separate PR as part of the work @suhanime is doing.

Reply:
I've removed it.